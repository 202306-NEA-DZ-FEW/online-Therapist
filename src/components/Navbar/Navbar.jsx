import Link from "next/link";
import Image from "next/image";
import { useEffect, useState } from "react";
import { useTranslation } from "next-i18next";
import { useRouter } from "next/router";
import { UserAuth } from "@/context/AuthContext";

export default function Navbar() {
    const router = useRouter();
    const [navbar, setNavbar] = useState(false);
    const { t } = useTranslation("common");
    const language = router.locale;
    const { user, logOut } = UserAuth();

    useEffect(() => {
        document.body.dir = language == "ar" ? "rtl" : "ltr";
    }, [language]);

    const handleSignOut = async () => {
        try {
            await logOut();
        } catch (error) {
            console.log(error);
        }
    };

    return (
        <div>
            <nav className='bg-white md:h-20 md:pt-4  p-0 md:px-12 border-b'>
                <div className='justify-between px-4 mx-auto md:items-center md:flex md:px-8'>
                    <div>
                        <div className='flex items-center justify-between py-3  md:py-0 '>
                            {/* LOGO */}
                            <Link href='/'>
                                <Image
                                    src='/Images/logo.png'
                                    width={100}
                                    height={100}
                                    alt='logo'
                                />
                            </Link>
                            <div className='md:hidden flex items-center'>
                                <div className='group inline-block relative'>
                                    <button className=' flex items-center justify-center text-DarkTeal font-atkinson text-lg text-center py-2  w-20 mx-auto  hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal'>
                                        <svg
                                            fill='none'
                                            height='24'
                                            viewBox='0 0 24 24'
                                            width='24'
                                            xmlns='http://www.w3.org/2000/svg'
                                        >
                                            <path
                                                clip-rule='evenodd'
                                                d='M12 4C7.58172 4 4 7.58172 4 12C4 16.4183 7.58172 20 12 20C16.4183 20 20 16.4183 20 12C20 7.58172 16.4183 4 12 4ZM2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12Z'
                                                fill='black'
                                                fill-rule='evenodd'
                                            />
                                            <path
                                                clip-rule='evenodd'
                                                d='M21 13H3V11H21V13Z'
                                                fill='black'
                                                fill-rule='evenodd'
                                            />
                                            <path
                                                clip-rule='evenodd'
                                                d='M11.2925 20.2933C11.2927 20.2931 11.2929 20.2929 12 21C12.7071 21.7071 12.7073 21.7069 12.7076 21.7066L12.7096 21.7046L12.7135 21.7007L12.7256 21.6884C12.7355 21.6782 12.7492 21.6642 12.7663 21.6464C12.8005 21.6108 12.8486 21.5599 12.9085 21.4946C13.0284 21.3641 13.196 21.1754 13.3954 20.9344C13.7935 20.4533 14.3224 19.7591 14.8517 18.8991C15.9014 17.1932 17 14.7582 17 12C17 9.24185 15.9014 6.8068 14.8517 5.1009C14.3224 4.24086 13.7935 3.54666 13.3954 3.06556C13.196 2.82458 13.0284 2.63589 12.9085 2.50537C12.8486 2.44008 12.8005 2.38925 12.7663 2.35361C12.7492 2.33579 12.7355 2.32176 12.7256 2.31161L12.7135 2.29931L12.7096 2.29536L12.7082 2.29394C12.7079 2.2937 12.7071 2.29289 12 3C11.2929 3.70711 11.2927 3.70691 11.2925 3.70672L11.2973 3.71164L11.3235 3.73867C11.3479 3.76407 11.3858 3.80406 11.4352 3.85791C11.5341 3.96567 11.679 4.12854 11.8546 4.34069C12.2065 4.76584 12.6776 5.38414 13.1483 6.1491C14.0986 7.6932 15 9.75815 15 12C15 14.2418 14.0986 16.3068 13.1483 17.8509C12.6776 18.6159 12.2065 19.2342 11.8546 19.6593C11.679 19.8715 11.5342 20.0343 11.4352 20.1421C11.3858 20.1959 11.3479 20.2359 11.3235 20.2613L11.2973 20.2884L11.2925 20.2933Z'
                                                fill='black'
                                                fill-rule='evenodd'
                                            />
                                            <path
                                                clip-rule='evenodd'
                                                d='M12.7075 20.2933C12.7073 20.2931 12.7071 20.2929 12 21C11.2929 21.7071 11.2927 21.7069 11.2924 21.7066L11.2904 21.7046L11.2865 21.7007L11.2744 21.6884C11.2645 21.6782 11.2508 21.6642 11.2337 21.6464C11.1995 21.6108 11.1514 21.5599 11.0915 21.4946C10.9716 21.3641 10.804 21.1754 10.6046 20.9344C10.2065 20.4533 9.6776 19.7591 9.14834 18.8991C8.09856 17.1932 7 14.7582 7 12C7 9.24185 8.09856 6.8068 9.14834 5.1009C9.6776 4.24086 10.2065 3.54666 10.6046 3.06556C10.804 2.82458 10.9716 2.63589 11.0915 2.50537C11.1514 2.44008 11.1995 2.38925 11.2337 2.35361C11.2508 2.33579 11.2645 2.32176 11.2744 2.31161L11.2865 2.29931L11.2904 2.29536L11.2918 2.29394C11.2921 2.2937 11.2929 2.29289 12 3C12.7071 3.70711 12.7073 3.70691 12.7075 3.70672L12.7027 3.71164L12.6765 3.73867C12.6521 3.76407 12.6142 3.80406 12.5648 3.85791C12.4659 3.96567 12.321 4.12854 12.1454 4.34069C11.7935 4.76584 11.3224 5.38414 10.8517 6.1491C9.90144 7.6932 9 9.75815 9 12C9 14.2418 9.90144 16.3068 10.8517 17.8509C11.3224 18.6159 11.7935 19.2342 12.1454 19.6593C12.321 19.8715 12.4658 20.0343 12.5648 20.1421C12.6142 20.1959 12.6521 20.2359 12.6765 20.2613L12.7027 20.2884L12.7075 20.2933Z'
                                                fill='black'
                                                fill-rule='evenodd'
                                            />
                                        </svg>
                                    </button>

                                    <ul className='absolute hidden w-full text-center md:text-start rounded-b-lg p-2 mt-0 bg-white shadow-lg text-DarkTeal group-hover:block md:w-24 '>
                                        <div className='space-y-2 z-50 '>
                                            <Link
                                                href='#'
                                                locale='en'
                                                className='hover:text-Teal'
                                            >
                                                <li className='w-full hover:bg-gray-50'>
                                                    English
                                                </li>
                                            </Link>
                                            <Link
                                                href='#'
                                                locale='ar'
                                                className='hover:text-Teal'
                                            >
                                                <li className='w-full hover:bg-gray-50'>
                                                    العربية
                                                </li>
                                            </Link>
                                        </div>
                                    </ul>
                                </div>
                                <button
                                    className='p-2 text-gray-700 rounded-md outline-none focus:border-gray-400 focus:border'
                                    onClick={() => setNavbar(!navbar)}
                                >
                                    {navbar ? (
                                        <Image
                                            src='/Images/close.svg'
                                            width={30}
                                            height={30}
                                            alt='logo'
                                        />
                                    ) : (
                                        <Image
                                            src='/Images/hamburger-menu.svg'
                                            width={30}
                                            height={30}
                                            alt='logo'
                                            className='focus:border-none active:border-none'
                                        />
                                    )}
                                </button>
                            </div>
                        </div>
                    </div>
                    <div>
                        <div
                            className={`flex-1 justify-self-center pb-3 mt-0 md:block md:pb-0 md:mt-0 ${
                                navbar ? "p-12 md:p-0 block" : "hidden"
                            }`}
                        >
                            <ul className='h-screen md:h-auto space-y-2 md:space-y-0 md:space-x-10 items-center justify-center md:flex md:justify-between md:items-center'>
                                <li
                                    className={`block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-96 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal  ${
                                        language === "ar" ? "pl-9" : ""
                                    }`}
                                >
                                    <Link
                                        href='/'
                                        onClick={() => setNavbar(!navbar)}
                                    >
                                        {t("navbar.home")}
                                    </Link>
                                </li>
                                <li className='block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-96 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal'>
                                    <Link
                                        href='/blogs'
                                        onClick={() => setNavbar(!navbar)}
                                    >
                                        {t("navbar.blogs")}
                                    </Link>
                                </li>
                                <li className='text-DarkTeal font-atkinson ml-2 text-lg md:hover:text-Teal z-50'>
                                    <div className='group inline-block relative'>
                                        <button className='block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-96 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal'>
                                            {t("navbar.about")}
                                        </button>

                                        <ul className='absolute hidden w-full text-center md:text-start rounded-b-lg p-2 mt-0 bg-white shadow-lg text-DarkTeal group-hover:block md:w-24'>
                                            <div className='space-y-2 '>
                                                <li className='w-full hover:bg-gray-50'>
                                                    <Link
                                                        href='/about-us'
                                                        className='hover:text-Teal'
                                                    >
                                                        {t("navbar.aboutUs")}
                                                    </Link>
                                                </li>
                                                <li className='w-full hover:bg-gray-50'>
                                                    <Link
                                                        href='/'
                                                        className='hover:text-Teal'
                                                    >
                                                        {t("navbar.team")}
                                                    </Link>
                                                </li>
                                                <li className='w-full hover:bg-gray-50'>
                                                    <Link
                                                        href='/careers'
                                                        className='hover:text-Teal'
                                                    >
                                                        {t("navbar.careers")}
                                                    </Link>
                                                </li>
                                            </div>
                                        </ul>
                                    </div>
                                </li>
                                <li className='block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-96 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal'>
                                    <Link
                                        href='/contact'
                                        onClick={() => setNavbar(!navbar)}
                                    >
                                        {t("navbar.contact")}
                                    </Link>
                                </li>
                                {!user ? (
                                    <li className='text-center'>
                                        <Link href='/login'>
                                            <button className='text-white font-atkinson text-lg bg-Teal rounded-full h-12 w-96 md:w-24 md:rounded-md md:h-10 hover:bg-DarkTeal'>
                                                {t("navbar.login")}
                                            </button>
                                        </Link>
                                    </li>
                                ) : (
                                    <div className='flex items-center space-x-4'>
                                        <Link href='/userProfile'>
                                            <p>Profile</p>
                                        </Link>
                                        <button
                                            onClick={handleSignOut}
                                            className='text-white font-atkinson text-lg bg-Teal rounded-full h-12 w-96 md:w-24 md:rounded-md md:h-10 hover:bg-DarkTeal'
                                        >
                                            Log Out
                                        </button>
                                    </div>
                                )}
                                <li className='text-DarkTeal font-atkinson ml-2 text-lg md:hover:text-Teal'>
                                    <div className='group inline-block relative'>
                                        <button className=' invisible md:visible flex items-center justify-center mt-2 text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-20 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal'>
                                            <svg
                                                fill='none'
                                                height='24'
                                                viewBox='0 0 24 24'
                                                width='24'
                                                xmlns='http://www.w3.org/2000/svg'
                                            >
                                                <path
                                                    clip-rule='evenodd'
                                                    d='M12 4C7.58172 4 4 7.58172 4 12C4 16.4183 7.58172 20 12 20C16.4183 20 20 16.4183 20 12C20 7.58172 16.4183 4 12 4ZM2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12Z'
                                                    fill='black'
                                                    fill-rule='evenodd'
                                                />
                                                <path
                                                    clip-rule='evenodd'
                                                    d='M21 13H3V11H21V13Z'
                                                    fill='black'
                                                    fill-rule='evenodd'
                                                />
                                                <path
                                                    clip-rule='evenodd'
                                                    d='M11.2925 20.2933C11.2927 20.2931 11.2929 20.2929 12 21C12.7071 21.7071 12.7073 21.7069 12.7076 21.7066L12.7096 21.7046L12.7135 21.7007L12.7256 21.6884C12.7355 21.6782 12.7492 21.6642 12.7663 21.6464C12.8005 21.6108 12.8486 21.5599 12.9085 21.4946C13.0284 21.3641 13.196 21.1754 13.3954 20.9344C13.7935 20.4533 14.3224 19.7591 14.8517 18.8991C15.9014 17.1932 17 14.7582 17 12C17 9.24185 15.9014 6.8068 14.8517 5.1009C14.3224 4.24086 13.7935 3.54666 13.3954 3.06556C13.196 2.82458 13.0284 2.63589 12.9085 2.50537C12.8486 2.44008 12.8005 2.38925 12.7663 2.35361C12.7492 2.33579 12.7355 2.32176 12.7256 2.31161L12.7135 2.29931L12.7096 2.29536L12.7082 2.29394C12.7079 2.2937 12.7071 2.29289 12 3C11.2929 3.70711 11.2927 3.70691 11.2925 3.70672L11.2973 3.71164L11.3235 3.73867C11.3479 3.76407 11.3858 3.80406 11.4352 3.85791C11.5341 3.96567 11.679 4.12854 11.8546 4.34069C12.2065 4.76584 12.6776 5.38414 13.1483 6.1491C14.0986 7.6932 15 9.75815 15 12C15 14.2418 14.0986 16.3068 13.1483 17.8509C12.6776 18.6159 12.2065 19.2342 11.8546 19.6593C11.679 19.8715 11.5342 20.0343 11.4352 20.1421C11.3858 20.1959 11.3479 20.2359 11.3235 20.2613L11.2973 20.2884L11.2925 20.2933Z'
                                                    fill='black'
                                                    fill-rule='evenodd'
                                                />
                                                <path
                                                    clip-rule='evenodd'
                                                    d='M12.7075 20.2933C12.7073 20.2931 12.7071 20.2929 12 21C11.2929 21.7071 11.2927 21.7069 11.2924 21.7066L11.2904 21.7046L11.2865 21.7007L11.2744 21.6884C11.2645 21.6782 11.2508 21.6642 11.2337 21.6464C11.1995 21.6108 11.1514 21.5599 11.0915 21.4946C10.9716 21.3641 10.804 21.1754 10.6046 20.9344C10.2065 20.4533 9.6776 19.7591 9.14834 18.8991C8.09856 17.1932 7 14.7582 7 12C7 9.24185 8.09856 6.8068 9.14834 5.1009C9.6776 4.24086 10.2065 3.54666 10.6046 3.06556C10.804 2.82458 10.9716 2.63589 11.0915 2.50537C11.1514 2.44008 11.1995 2.38925 11.2337 2.35361C11.2508 2.33579 11.2645 2.32176 11.2744 2.31161L11.2865 2.29931L11.2904 2.29536L11.2918 2.29394C11.2921 2.2937 11.2929 2.29289 12 3C12.7071 3.70711 12.7073 3.70691 12.7075 3.70672L12.7027 3.71164L12.6765 3.73867C12.6521 3.76407 12.6142 3.80406 12.5648 3.85791C12.4659 3.96567 12.321 4.12854 12.1454 4.34069C11.7935 4.76584 11.3224 5.38414 10.8517 6.1491C9.90144 7.6932 9 9.75815 9 12C9 14.2418 9.90144 16.3068 10.8517 17.8509C11.3224 18.6159 11.7935 19.2342 12.1454 19.6593C12.321 19.8715 12.4658 20.0343 12.5648 20.1421C12.6142 20.1959 12.6521 20.2359 12.6765 20.2613L12.7027 20.2884L12.7075 20.2933Z'
                                                    fill='black'
                                                    fill-rule='evenodd'
                                                />
                                            </svg>
                                        </button>

                                        <ul className='absolute hidden w-full text-center md:text-start rounded-b-lg p-2 mt-0 bg-white shadow-lg text-DarkTeal group-hover:block md:w-24'>
                                            <div className='space-y-2 '>
                                                <Link
                                                    href='#'
                                                    locale='en'
                                                    className='hover:text-Teal'
                                                >
                                                    <li className='w-full hover:bg-gray-50'>
                                                        English
                                                    </li>
                                                </Link>
                                                <Link
                                                    href='#'
                                                    locale='ar'
                                                    className='hover:text-Teal'
                                                >
                                                    <li className='w-full hover:bg-gray-50'>
                                                        العربية
                                                    </li>
                                                </Link>
                                            </div>
                                        </ul>
                                    </div>
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
            </nav>
        </div>
    );
}
