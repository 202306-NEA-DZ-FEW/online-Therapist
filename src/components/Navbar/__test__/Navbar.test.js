import renderer from "react-test-renderer";
import Navbar from "../Navbar";

it("renders correctly", () => {
    // const authInstance = require("firebase/auth").getAuth();
    // Mock methods or properties of authInstance as needed

    const tree = renderer.create(<Navbar />).toJSON();
    expect(tree).toMatchSnapshot();
});
