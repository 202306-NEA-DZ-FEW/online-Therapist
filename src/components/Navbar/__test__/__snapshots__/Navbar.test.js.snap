// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="z-50 sticky top-0 bg-white "
  style={Object {}}
>
  <div
    className=" mx-0 md:container md:mx-auto "
  >
    <div
      className="relative justify-between bg-white md:items-center md:flex md:px-8 h-20 "
    >
      <div
        className=""
      >
        <div
          className="flex items-center justify-between py-3 md:py-0 "
        >
          <a
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <img
              alt="logo"
              data-nimg="1"
              decoding="async"
              height={100}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=%2FImages%2Flogo.png&w=256&q=75"
              srcSet="/_next/image?url=%2FImages%2Flogo.png&w=128&q=75 1x, /_next/image?url=%2FImages%2Flogo.png&w=256&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={100}
            />
          </a>
          <div
            className="md:hidden flex gap-x-4 items-center"
          >
            <div
              className="flex justify-end space-y-[30px] md:justify-center md:items-center md:flex-col md:space-y-[300px] z-50"
            >
              <div
                className="text-3xl relative"
                onClick={[Function]}
              >
                <svg
                  fill="none"
                  height="1em"
                  stroke="currentColor"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                    stroke="none"
                  />
                  <path
                    d="M12 12m-9 0a9 9 0 1 0 18 0a9 9 0 1 0 -18 0"
                  />
                  <path
                    d="M12 10m-3 0a3 3 0 1 0 6 0a3 3 0 1 0 -6 0"
                  />
                  <path
                    d="M6.168 18.849a4 4 0 0 1 3.832 -2.849h4a4 4 0 0 1 3.834 2.855"
                  />
                </svg>
              </div>
              <div
                className="absolute "
              />
            </div>
            <button
              className="p-2 text-gray-700 rounded-md outline-none focus:border-gray-400 focus:border"
              onClick={[Function]}
            >
              <svg
                aria-hidden="true"
                className="text-3xl text-black p-0 m-0"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 20 20"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clipRule="evenodd"
                  d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
                  fillRule="evenodd"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div>
        <div
          className="md:block hidden"
        >
          <ul
            className="bg-white h-screen md:mt-0 md:h-auto space-y-2 md:space-y-0 md:space-x-10  items-center md:justify-center md:flex justify-start md:items-center flex-col flex md:flex-row pt-10 md:pt-0"
          >
            <li
              className="block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-48 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto  md:hover:border-b md:hover:text-Teal  "
            >
              <a
                href="/"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                navbar.home
              </a>
            </li>
            <li
              className="block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-48 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal"
            >
              <a
                href="/blogs"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                navbar.blogs
              </a>
            </li>
            <li
              className="block text-DarkTeal font-atkinson text-lg text-center md:text-start py-2 rounded-full w-48 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal "
            >
              <div
                className="group inline-block relative flex justify-center items-center flex-col space-y-[180px] "
              >
                <button
                  className="block flex justify-center items-center "
                >
                  navbar.about
                   
                  <svg
                    className="text-xl"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    strokeWidth="0"
                    style={
                      Object {
                        "color": undefined,
                      }
                    }
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                </button>
                <div
                  className="absolute hidden bg-white z-50 group-hover:block "
                >
                  <div
                    className="md:w-36 w-48 bg-white font-atkinson border p-2 shadow-md rounded-lg "
                  >
                    <ul
                      className="py-2 first:pt-0 z-60 last:pb-0"
                    >
                      <a
                        href="/about-us"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          navbar.aboutUs
                        </li>
                      </a>
                      <a
                        href="/team"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          navbar.team
                        </li>
                      </a>
                      <a
                        href="/careers"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          navbar.careers
                        </li>
                      </a>
                    </ul>
                  </div>
                </div>
              </div>
            </li>
            <li
              className="block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-48 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal"
            >
              <a
                href="/contact"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                navbar.contact
              </a>
            </li>
            <div
              className="hidden md:flex justify-center items-center flex-col space-y-[300px] z-50"
            >
              <div
                className="text-3xl relative"
                onClick={[Function]}
              >
                <img
                  alt="Profile"
                  className="w-8 object-cover h-8 rounded-full"
                />
              </div>
              <div
                className="absolute outline-none"
              />
            </div>
            <li
              className="hidden md:flex"
            >
              <div
                className="group inline-block relative flex justify-center items-center flex-col space-y-[150px] z-50"
              >
                <button
                  className="flex items-center justify-center mt-2 text-xl py-2 w-20 mx-auto md:w-auto "
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    strokeWidth="0"
                    style={
                      Object {
                        "color": undefined,
                      }
                    }
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M478.33 433.6l-90-218a22 22 0 00-40.67 0l-90 218a22 22 0 1040.67 16.79L316.66 406h102.67l18.33 44.39A22 22 0 00458 464a22 22 0 0020.32-30.4zM334.83 362L368 281.65 401.17 362zm-66.99-19.08a22 22 0 00-4.89-30.7c-.2-.15-15-11.13-36.49-34.73 39.65-53.68 62.11-114.75 71.27-143.49H330a22 22 0 000-44H214V70a22 22 0 00-44 0v20H54a22 22 0 000 44h197.25c-9.52 26.95-27.05 69.5-53.79 108.36-31.41-41.68-43.08-68.65-43.17-68.87a22 22 0 00-40.58 17c.58 1.38 14.55 34.23 52.86 83.93.92 1.19 1.83 2.35 2.74 3.51-39.24 44.35-77.74 71.86-93.85 80.74a22 22 0 1021.07 38.63c2.16-1.18 48.6-26.89 101.63-85.59 22.52 24.08 38 35.44 38.93 36.1a22 22 0 0030.75-4.9z"
                    />
                  </svg>
                </button>
                <div
                  className="absolute hidden group-hover:block"
                >
                  <div
                    className="w-48 md:w-32 bg-white font-atkinson border bg-grey p-2 shadow-md rounded-lg"
                  >
                    <ul
                      className="py-2 first:pt-0  last:pb-0"
                    >
                      <a
                        href="#"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          English
                        </li>
                      </a>
                      <a
                        href="#"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          العربية
                        </li>
                      </a>
                    </ul>
                  </div>
                </div>
              </div>
            </li>
            <li
              className="md:hidden block text-DarkTeal font-atkinson text-lg text-center py-2 rounded-full w-48 mx-auto border border-gray-200 hover:bg-gray-50 hover:text-Teal md:border-0 md:border-b-[#1E4445] md:hover:bg-white md:rounded-none md:py-0 md:w-auto md:hover:border-b md:hover:text-Teal"
            >
              <div
                className="group  relative flex justify-center items-center flex-col space-y-[135px] md:space-y-[150px] "
              >
                <button
                  className=" md:block "
                >
                  navbar.language
                </button>
                <div
                  className="absolute hidden group-hover:block"
                >
                  <div
                    className="w-48 md:w-32 bg-white font-atkinson border bg-grey p-2 shadow-md rounded-lg"
                  >
                    <ul
                      className="py-2 first:pt-0  last:pb-0"
                    >
                      <a
                        href="#"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          English
                        </li>
                      </a>
                      <a
                        href="#"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onTouchStart={[Function]}
                      >
                        <li
                          className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
                          href="#"
                        >
                          العربية
                        </li>
                      </a>
                    </ul>
                  </div>
                </div>
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</nav>
`;
